packages/driver/src/cross-origin/patches/cookies.js

```diff
@@ -77,4 +136,30 @@ export const patchDocumentCookie = (Cypress, window) => {
   }
 
   window.addEventListener('unload', onUnload)
+
+  Cypress.on('window:before:load', patch)
```

Comment:
Are we patching here to ensure that state is setup first? We will have to come up with something else if there isn't a spec bridge attached.

Reply:
It's here because it needs to be done on before the page loads. If we move this into the injection it will just work since the injection is the first script run.

Reply:
oh, that makes sense.